### ISSUE TITLE
Symfony 6 compatibility

### ISSUE THREAD

**Issue by cbeyer** [Kunstmaan/KunstmaanBundlesCMS#3246]
Hi,

Could this bundle (and his "brothers", I use kunstmaan/admin-bundle and kunstmaan/media-bundle and ) be compatible with Symfony 6 ?

When looking https://packagist.org/packages/kunstmaan/bundles-cms#6.2.3 the bundle needs a lot of symfony bundles version ^4.4|^5.4 :(

**Comment by acrobat** [comment]
Symfony 6 support is being worked on but due to possible bc breaks on our side to be compatible with symfony 6 we will need to deprecate code so the CMS is ready for symfony 6 in cms v7. You can keep an eye out for PR's/issues with the [sf6-compatibility label](https://github.com/Kunstmaan/KunstmaanBundlesCMS/labels/sf6-compatibility) and the [symfony 6 project board](https://github.com/Kunstmaan/KunstmaanBundlesCMS/projects/10) to track the work

**Comment by OskarStark** [comment]
Hi @acrobat 👋 

Any news when Symfony 6 will be supported?

cc @silasjoisten @igst

**Comment by acrobat** [comment]
@OskarStark the 7.1 branch currently allows and supports Symfony 6. I was able to test this on a few projects and everything seems to work as expected, but if you can test this also on your projects and report any issues you encounter! A release should not be long but a few refactoring/deprecations need to be done on 6.x before I can tag the 7.0 and 7.1 releases!

**Comment by OskarStark** [comment]
Glad to hear.

Currently on holiday.

@silasjoisten can you try out the 7 branch please? Thanks

**Comment by OskarStark** [comment]
Is there any progress, because Symfony 7 is on it's way :-(

**Comment by cbeyer** [comment]
Up! ping @acrobat  ?

**Comment by acrobat** [comment]
sorry for the late reply but I still need to do 1 deprecation of a part of our code in 6.4 before we can release the 6.4 and 7.0 version of our CMS. After that 7.1 with symfony 6 support should come soon after.

The symfony 6 prep work I've already done on symfony 6.3 so I have to check for the new deprecations in symfony 6.4. In the meanwhile you are welcome to test the 7.1 branch and report any issues or remaining deprecations.

**Comment by OskarStark** [comment]
What about an RC with Sf 6.3 support?

**Comment by acrobat** [comment]
Yes that might be possible but first I need to finish release the latest 6.x and 7.0 release. I will try to make some time for this soon.

**Comment by acrobat** [comment]
7.1.0 will contain Symfony 6 support, I've tagged a 7.1.0-alpha2 release. Please test and report any issues, I want to tag the 7.1 stable release later this week

**Comment by acrobat** [comment]
[7.1.0](https://github.com/Kunstmaan/KunstmaanBundlesCMS/releases/tag/7.1.0) with symfony 6 support is released!
